designs/administrator/olm/operand-form-yaml-switching/index.md

```diff
@@ -1,26 +1,38 @@
 ---
 parent: Administrator
-version: 4.4
+version: 4.5
 ---
 
-# Operand (CR) Creation Form/YAML Switching
+# Operand (Custom Resource) Creation and Editing Form/YAML Switching
 
-The existing “Edit Form” and “Edit YAML” links have been changed to a toggle control to better convey that the user can freely switch between the two without losing changes between them. The control is also moved to immediately above the editors to aid discovery. The use of radios with a label also improves accessibility.
+The existing **Edit Form** and **Edit YAML** links that have been used to switch between views have been changed to a toggle control to better convey that the user can freely switch between the two without losing changes between them. The controls are also located immediately above the editors to aid discovery, and the use of radios with a label improves accessibility.
```

Comment:
@itsptk I think we need to note the pages where we use the Edit Yaml link in the top right corner today so a developer has those identified as pages they need to touch and update to this new toggle. 

From what I can tell it's only Routes, PVCs, and Storage Classes. Does that look correct to you?

Reply:
I think that is right, though this story is only affecting operators' custom resource creation and editing for 4.5, since those are the only resources that currently let you freely flip back and forth between the editors (and preserve the changes between each.) There will need to be follow on storie(s) to get those named resources to support freely switching back and forth and also using this radio interaction, today they are just 'one way streets' as far as switching to the YAML editor from form.

Reply:
@spadgett @benjaminapetersen @TheRealJon Would it be a significant amount of work to have these other (3) resources support switching back and forth between the editors? We will now have two different interactions for switching between forms and YAML, the 'Edit' link when one-way (routes, PVCs, storage classes) and the radios when freely switching back and forth (operands). Would we want to prioritize this soon to avoid this inconsistency? cc @alimobrem

Reply:
I think we are unlikely to have extra cycles in 4.5 to pull this off, but am in favor of continuing towards consistency in 4.6.

Reply:
@itsptk @beanh66 It would definitely take a bit of work to implement the toggling interaction on those three workflows. They each have a novel implementation for their respective forms, so we would need to evaluate them individually. As part of my work on the create operand workflow, I am trying to pull out some generic components in hopes that we can reuse them in the future. Hopefully, that will ease the transition for future workflows.

Reply:
Awesome, thanks @TheRealJon! @itsptk @benjaminapetersen Can we make Console Jira stories to track that follow on work in the backlog so we don't forget about updating those specific pages in 4.6?

Reply:
@TheRealJon This create operand code is shared with the dev console, correct? This change will appear there in 4.5 as well?

@serenamarie125 Is there anywhere else you expect to use this interaction in the dev console in 4.5? Pipelines was mentioned? This interaction will likely become more widespread in the admin console in 4.6.

Reply:
@itsptk Yes, the component is shared across both admin and dev consoles. We already have a little bit of conditional logic based on the perspective.

Reply:
> Awesome, thanks @TheRealJon! @itsptk @benjaminapetersen Can we make Console Jira stories to track that follow on work in the backlog so we don't forget about updating those specific pages in 4.6?

Created: https://issues.redhat.com/browse/CONSOLE-2147 & https://issues.redhat.com/browse/PD-441